common/src/lib.rs

```diff
@@ -378,7 +381,10 @@ pub enum DescriptionFrom {
 
 /// Function that can be performed when a dragged object is released
 /// on a drop target.
-/// Note: aria-dropeffect is deprecated in WAI-ARIA 1.1.
+///
+/// Note: [`aria-dropeffect`] is deprecated in WAI-ARIA 1.1.
+///
+/// [`aria-dropeffect`]: https://www.w3.org/TR/wai-aria-1.1/#aria-dropeffect
```

Comment:
Depending on how you want to present this to downstream users of the `accesskit` crate (who I presume correspond to "authors" in ARIA, vs the "user agents" which seems to match more closely to the platform adapters), we could use:
```suggestion
#[deprecated = "`aria-dropeffect` is deprecated in WAI-ARIA 1.1"]
```

The one downside of this is that the deprecation warnings trigger in this crate as well (this issue is tracked in https://github.com/rust-lang/rust/issues/47219). Most could be worked around with a local `#[allow(deprecated)]`, but I could not find any way to disable these for the `DropEffect` enum specifically other than a global `#![allow(deprecated)]`.